### ISSUE TITLE
Code preview feature

### ISSUE THREAD

**Issue by gustavoFreireS** [lauthieb/code-notes#18]
Hi and thanks for this well made, software

It would be nice if you add code preview option for stylesheets or markup languages (an easy implementation would be creating an iframe importing the files on view) in the case of preprocessed ones (like scss) you could use a javascript implementation of the preprocessor (like this one: https://github.com/medialize/sass.js/)

**Comment by lauthieb** [comment]
Hello @gustavoFreireS !

Thanks for this feature idea, I think this feature will be able in V1.3 because V1.2 is already quite complete.
I will keep you informed about the progression ;)

**Comment by gustavoFreireS** [comment]
thanks for the aswer, btw I could implement this change if you wnat

**Comment by lauthieb** [comment]
Feel free @gustavoFreireS, I would appreciate this :)

**Comment by lauthieb** [comment]
Hello @gustavoFreireS, do you plan to implement this please ? :)

**Comment by gustavoFreireS** [comment]
Yeah still on, will start doing it following a similar project of mine: https://github.com/gustavoFreireS/react-text-editor

**Comment by lauthieb** [comment]
@gustavoFreireS cool, thanks 😃
